analytics-services/src/main/resources/analytics-es-template.json

```diff
@@ -49,6 +49,10 @@
         "errorMessage": {
           "type": "text",
           "index_options": "offsets"
+        },
+        "profileProperties": {
+          "type": "object",
+          "dynamic": true
```

Comment:
as you change the index, did you take care of updating the existing index ? create a new version ?

Reply:
analytics dynamically creates new indexes each day, i don't think that this will cause any problem.

Reply:
we will check on experience instance